packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsScreen.tsx

```diff
@@ -0,0 +1,76 @@
+    const { data: isEuRegion } = useIsEuRegion();
+
+    const { is_mf: isEuUser = false, is_virtual: isDemo = false } = activeTrading || {};
+
+    const { data: compareAccounts, hasCTraderAccountAvailable, hasDxtradeAccountAvailable } = useCFDCompareAccounts();
+    const { data: cfdAccounts } = useCFDAccountsList();
+
+    const { ctraderAccount, dxtradeAccount, mt5Accounts } = compareAccounts;
+
+    const isDxtradeAdded = useMemo(
+        () => !!cfdAccounts && isDxtradeAccountAdded(cfdAccounts.dxtrade, isDemo),
+        [cfdAccounts, isDemo]
+    );
+
+    const isCtraderAdded = useMemo(
+        () => !!cfdAccounts && isCTraderAccountAdded(cfdAccounts.ctrader, isDemo),
+        [cfdAccounts, isDemo]
+    );
+
+    return (
+        <div className='overflow-x-auto overflow-y-scroll lg:w-full lg:h-full m-50 max-h-[800px] pt-50 px-50 pb-[201px]'>
+            <CompareAccountsHeader isDemo={isDemo} isEuRegion={isEuRegion} />
+            <div className='flex justify-center p-1000 lg:my-50 lg:mx-auto max-w-[1232px]'>
+                <CompareAccountsCarousel>
+                    {mt5Accounts?.map(item => (
+                        <CFDCompareAccountsCard
+                            isAccountAdded={item?.is_added}
+                            isDemo={isDemo}
+                            isEuRegion={isEuRegion}
+                            isEuUser={isEuUser}
+                            key={`${item?.market_type} ${item?.shortcode}`}
+                            marketType={item?.market_type}
+                            platform={item?.platform}
+                            shortCode={item?.shortcode}
+                        />
+                    ))}
+                    {/* Renders cTrader data */}
+                    {mt5Accounts?.length && hasCTraderAccountAvailable && ctraderAccount && (
+                        <CFDCompareAccountsCard
+                            isAccountAdded={isCtraderAdded}
+                            isDemo={isDemo}
+                            isEuRegion={isEuRegion}
+                            isEuUser={isEuUser}
+                            marketType={ctraderAccount.market_type}
+                            platform={ctraderAccount.platform}
+                            shortCode={ctraderAccount.shortcode}
+                        />
+                    )}
+                    {/* Renders Deriv X data */}
+                    {mt5Accounts?.length && hasDxtradeAccountAvailable && dxtradeAccount && (
+                        <CFDCompareAccountsCard
+                            isAccountAdded={isDxtradeAdded}
+                            isDemo={isDemo}
+                            isEuRegion={isEuRegion}
+                            isEuUser={isEuUser}
+                            marketType={dxtradeAccount.market_type}
+                            platform={dxtradeAccount.platform}
+                            shortCode={dxtradeAccount.shortcode}
```

Comment:
Question: can't we access any of the stuff directly in the card. Is there a possibility to that?

Reply:
No every thing should be passed as Different prop like market, platform etc.